components/uikit/Input/TextareaInput.tsx

```diff
@@ -0,0 +1,76 @@
+import {
+  Box,
+  FormControl,
+  FormControlError,
+  FormControlErrorIcon,
+  FormControlErrorText,
+  FormControlHelper,
+  FormControlHelperText,
+  Textarea,
+  TextareaInput,
+} from "@gluestack-ui/themed";
+
+export type TextareaInputPropos = {
+  label: string,
+  placeholder: string;
+  error?: string;
+  help?: string;
+  value?: string;
+  onChange?: (value: string) => void;
+  isDisabled: boolean;
+  isRequired: boolean;
+};
+
+export const TextAreaInput = ({
+  label,
+  placeholder,
+  error,
+  help,
+  value,
+  onChange,
+  isDisabled,
+  isRequired,
+}: TextareaInputPropos) => {
+  const isInvalid = error ? true : false;
+  return (
+    <Box>
+      <FormControl
+        isInvalid={isInvalid}
+        isRequired={isRequired}
+        isDisabled={isDisabled}
+      >
+        <Textarea
+          p="$4"
+          h={176}
+          borderRadius="$xl"
+          borderWidth={1.5}
+          borderColor="$white"
+          $focus-borderColor="$primaryDark"
+        >
+          <TextareaInput
+            placeholder={placeholder}
+            fontSize="$md"
+            p="$0"
+            placeholderTextColor="$primaryNormal"
+            color="$primaryNormal"
+            value={value}
+            onChangeText={onChange}
+          />
+        </Textarea>
+        {help ? (
+          <FormControlHelper>
+            <FormControlHelperText fontSize="$sm" color="$black">
+              {help}
+            </FormControlHelperText>
+          </FormControlHelper>
+        ) : null}
+        {isInvalid ? (
+          <FormControlError>
+            {/* <FormControlErrorIcon as={AlertCircleIcon} color="$red"/> */}
+            <FormControlErrorText fontSize="$sm">{error}</FormControlErrorText>
+          </FormControlError>
+        ) : null}
```

Comment:
This component has common parts with the  TextInput component. These part includes the lines `60` to `72` and the lines `27` to `31`, so the most structured implementation should refactor the TextInput to create a wrapper component that hold these common lines.

Reply:
I will merge it, but note that you have to do the refactor latter.